web/libs/editor/src/tags/object/TimeSeries/ChannelLegend.module.scss

```diff
@@ -0,0 +1,31 @@
+.channelLegend {
+  --marker-size: 10px;
+  --marker-color:  var(--color-primary-surface, #4C5FA9);
+  --marker-hidden-color: var(--color-neutral-surface, #F9F8F6);
```

Comment:
It's better to not default semantic tokens as we won't know up front if the correct dark mode compatible colours would be applying.

```suggestion
  --marker-color:  var(--color-primary-surface);
  --marker-hidden-color: var(--color-neutral-surface);
```